### DISCUSSION TITLE
Connection failure for Postgres

### DISCUSSION THREAD

**Discussion by nick-nea** [nocodb/nocodb#4032] [Q&A]
I'm having trouble connecting to my local Postgres instance. It keeps giving me the same error:

![image](https://user-images.githubusercontent.com/115644466/195368421-a105a30a-9b97-4a8d-8fdb-7c2c7002a94c.png)

![image](https://user-images.githubusercontent.com/115644466/195369027-c78fd017-ba5d-4846-a61b-d804cd24109f.png)

I've tried adding new roles, different passwords.  When logging in via `psql`, it works just fine.  

Setup: 
- Mac
- NocoDB Installed via homebrew
- Postgres installed in Docker Desktop K8s exposed via NodePort
- No SSL

Thoughts - Should I try using SSL?  Not sure where the SSL certs would be though.  

I appreciate any help or advice. Really interested in using this tool!

**Comment by o1lab** [comment]
<img width="827" alt="Screenshot 2022-10-12 at 16 01 35" src="https://user-images.githubusercontent.com/5435402/195378386-ef9dbce4-c012-44a4-afe5-412af44949c9.png">

Can you try SSL to set allowed or preferred ?

**Reply by nick-nea** [reply]
Yes, but where do I find the certs?

**Reply by o1lab** [reply]
certs are not required. Only set preferred / allowed. It sets certain flags.
